### PR TITLE
@mzikherman => [WorksForYou] Add <MarketingHeader /> component

### PR DESCRIPTION
**Pull request by damassi**
This adds the (currently WIP, from design perspective) `<MarketingHeader />` component to the Works For You page: 

screen shot 2018-08-27 at 7 33 41 pm

@mzikherman - should this be gated behind a Feature Flag? or is it already gated at the Force level?

### CODE REVIEW

File: src/Styleguide/Elements/Box.tsx [hunk 1]
```diff
@@ -36,22 +36,24 @@ import {
...
 
 export const BorderBox = styled(Flex).attrs<BorderBoxProps>({})`
   border: 1px solid ${color("black10")};
   border-radius: 2px;
   padding: ${space(2)}px;
-  ${styledSpace};
+  ${background};
+  ${height};
   ${maxWidth};
+  ${styledSpace};
```
Comment (damassi) [inline src/Styleguide/Elements/Box.tsx]: The following on down is all alpha

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit cb863023af9e21209935e02665752d357e4c785a

https://deploy-preview-1190--artsy-reaction.netlify.com

**Comment by mzikherman** [comment]
Question- we're still under a lab feature so this is fine to merge, but theoretically, how would we work with WIP/iterative PR's to already deployed components? 

For instance, someone else might bump Reaction in Force later today, which would deploy this before it's ready. Ran into this myself, so am wondering!

**Comment by damassi** [comment]
@mzikherman - yah, i didn't expect this to be merged yet as i had a question in the description; I should have pasted it above the image!  

> should this be gated behind a Feature Flag? or is it already gated at the Force level?

**Comment by mzikherman** [comment]
haha whoops!

well then yea- this whole component is gated behind a lab feature, so 👍

**Pull request merged**